climada/test/test_api_client.py

```diff
@@ -144,7 +146,6 @@ def test_get_hazard(self):
         self.assertEqual(np.max(hazard.centroids.region_id), 756)
```

Comment:
Should not more than one hazard instance be returned now since concatenation is not part of `get_hazard` anymore?

Reply:
👍 right. the test hasn't been fixed - until now